docs/reference/extensions/flask-framework.rst

```diff
@@ -16,6 +16,12 @@ server metrics.
     The Flask extension is compatible with the ``bare``, ``ubuntu@22.04``
     and ``ubuntu@24.04`` bases.
 
+The Flask extension supports both synchronous and asynchronous
+Gunicorn workers. If you want asynchronous workers, you have to add
+``gevent`` package to the ``requirements.txt`` file.
+Read more
+:ref:`Using asynchronous Gunicorn workers <flask-async-gunicorn-workers>`.
```

Comment:
Since it discusses dependencies, I think the second sentence should be added as an option in the _Project requirements_, section, like so:

```rst
2. The project must...

For the project to make use of asynchronous Gunicorn workers:

- The ``requirements.txt`` file must include ``gevent`` as a dependency.
```

Reply:
Hey @medubelko thanks for your comments? Should I move the `Gunicorn workers. If you want asynchronous workers, you have to add
``gevent`` package to the ``requirements.txt`` file.` part to _Project requirements_ section or copy it there?